### ISSUE TITLE
missing tostring()

### ISSUE THREAD

**Issue by daurnimator** [simoncozens/cassowary.lua#4]
Gets an error when tracing is enabled.

```
lua: /usr/share/lua/5.3/cassowary.lua:684: attempt to concatenate a table value (field '?')
stack traceback:
    /usr/share/lua/5.3/cassowary.lua:684: in method 'initializer'
    /usr/share/lua/5.3/cassowary.lua:662: in field '_init'
    /usr/share/lua/5.3/std/container.lua:174: in function </usr/share/lua/5.3/std/container.lua:156>
    (...tail calls...)
    main.lua:5: in main chunk
    [C]: in ?
```

Currently the line is:

``` lua
      cassowary:tracePrint("objective expr == "..self.rows[self.objective])
```

It should be this instead:

``` lua
      cassowary:tracePrint("objective expr == "..tostring(self.rows[self.objective]))
```

**Comment by simoncozens** [comment]
This actually got fixed with the penlight migration in 32ac9aef. Only four years late!

**Comment by daurnimator** [comment]
Welp. Now I don't recall what I was even doing at the time :P
